src/xcode/ENA/.swiftlint.yml

```diff
@@ -226,3 +226,21 @@ trailing_whitespace:
 
 vertical_whitespace:
   max_empty_lines: 2
+  
+file_length:
+  warning: 600
+  error: 1200
```

Comment:
wow, quit big limits 

maybe this
  warning: 500
  error: 700

Reply:
I reduced the error to 700.
Would like to keep the warning at 500 since we have a lot of files around ~440 Lines.

Is that fine with you?

Reply:
for me it's fine